requirements/requirements-3.10-Linux.txt

```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile with Python 3.10
 # by the following command:
 #
-#    pip-compile --allow-unsafe --generate-hashes --output-file=requirements-3.10-Linux.txt base.in dev.in gui-dev.in gui.in linters.in
```

Comment:
I think these requirements compiles need to be re-run now, the `base.universal2.in` should no longer be included.